DQM/L1TMonitor/src/L1TdeRCT.cc

```diff
@@ -435,11 +433,11 @@ if(verbose_) std::cout << " ECAL data: Energy: " << iEcalTp->compressedEt() << "
   bool regionEmulQuiet    [PhiEtaMax] = {false};
   bool regionEmulHfPlusTau[PhiEtaMax] = {false};
 
-if(first)
-{
-  first = false ;
-}
-
+  if(first)
```

Comment:
If this is the only place first is used in the file, I think it can be just dropped :).